contents/product-engineers/what-is-user-stickiness.md

```diff
@@ -0,0 +1,120 @@
+
+### Are you a HAU, DAU, WAU, MAU, QAU, or YAU?
+
+How often do people use your product? The answer to that will depend on what kind of product or service you’ve built. A product can still be ‘sticky’ even if it’s used just once a week or once a month provided that’s how you expect people to interact with it.
+
+Think of it like this: what problem is your product solving? And how often do people need to solve that problem? 
+
+At Uber, for example, ‘normal’ usage might be for a user to book a ride twice a month. At Airbnb, it might be ‘normal’ for someone to book a trip once per year. It’s unlikely that a ‘normal’ PayPal customer uses the service every week, but they likely use it every month.
+
+Meta’s products and services have become so dominant because they’re very good at getting users hooked every day, but this simply isn’t possible, or necessary, in a lot of industries. Users of B2B SaaS products, for example, often have no choice but to use the products their employer chooses. Here, metrics like adoption, retention, feature use, and net promoter score are the [most important metrics](https://posthog.com/blog/b2b-saas-product-metrics).
+
+But how do you work out where your company sits? According to research by [Sequoia](https://medium.com/sequoia-capital/selecting-the-right-user-metric-de95015aa38), the threshold for stickiness is 50% for HAU/DAU, and 60% for DAU/WAU, WAU/MAU and MAU/QAU and QAU/YAU. Think of it like this table:
+
+[TABLE HERE](https://miro.medium.com/v2/resize:fit:1312/format:webp/0*uTjliTsC7Gusreg-)
+
+If you measure your HAU/DAU and it’s 50% or above, then your product is used daily and you should focus on your HAU/DAU. If your HAU/DAU is below 50% then keep reading right across the table until you pass one of the thresholds. If your DAU/WAU is 68%, for example, then focus on your DAU/WAU. If it’s only 30% then keep going right until you pass one of the thresholds.
+
+This table is also a good way of identifying different kinds of users. If your WAU/MAU is 65% but your DAU/MAU is 20% then that still means there’s a significant subset of users actively using your product every day – even if you consider a ‘normal’ user as someone who is only active once a week.
+
+### What’s a good DAU/MAU ratio?
+
+This is dependent on a lot of factors. What is your product? What industry do you operate in? Who is the intended audience? What is your business model? Facebook, the granddaddy of the DAU/MAU ratio, has almost always had a ratio of 50% or more. It recently stopped reporting this figure in its financial results, but back in March 2023 it was 68%. That’s incredibly high – and a big reason why parent company Meta continues to be so valuable. 
+
+Most products and services can only dream of such numbers. According to research by [Mixpanel](https://mixpanel.com/blog/2024-mixpanel-benchmarks-report/), the average DAU/MAU ratio across all industries in 2023 was 37%. Here’s how that breaks down by industry:
```

Comment:
I would slightly avoid linking to a competitor unless you absolutely need to

Reply:
Good benchmarks numbers for this are pretty hard to find. Happy to not link out to Mixpanel, but I struggled to find a better source. Will take another look and we can make a call.